### ISSUE TITLE
Add ForceKextsToLoad feature to OpenCore

### ISSUE THREAD

**Issue by chriswayg** [acidanthera/bugtracker#681]
In Clover there is  [ForceKextsToLoad](https://sourceforge.net/p/cloverefiboot/wiki/KernelAndKextPatches/#forcekextstoload), but OpenCore does not seem to offer this feature. 

This has been asked about in a few places (r/hackintosh, insanelymac) without an answer, so it seems to me, that it has not been implemented yet in OpenCore.

Example:
```
<key>ForceKextsToLoad</key>
<array>
  <string>\System\Library\Extensions\IO80211Family.kext</string>
</array>
```
Force load kexts (and plugins) from booted volume.

**Comment by vit9696** [comment]
Hello, we indeed do not have this feature, but the reason for this is that it does not always work. For example, ForceKextsToLoad in Clover does not work at all with FileVault 2 enabled, as it makes /System/Library/Extensions directory unaccessible for the bootloader. We believe FileVault 2 should be used by everyone, so this feature is not an option for us.

We plan to implement an alternative solution for this problem: Apple Secure Boot. This way immutablekernel will be used instead of prelinkedkernel, which contains all the drivers.

For now your only solution might be to inject IO80211 as well. Closing as a duplicate of #368
